nbzip/static/tree.js

```diff
@@ -0,0 +1,62 @@
+
+
+define([
+    'jquery',
+    'base/js/utils',
+    'base/js/namespace'
+], function (
+    $, utils, Jupyter
+) {
+
+    // inspired by: https://stackoverflow.com/questions/1106377/detect-when-browser-receives-file-download
+    currToken = null;
```

Comment:
var, otherwise this becomes a global variable :)

Reply:
ok, sounds good. thank you for the catch.